src/drv_oculus_rift/rift.c

```diff
@@ -315,6 +315,30 @@ static ohmd_device* open_device(ohmd_driver* driver, ohmd_device_desc* desc)
 	priv->base.properties.lens_vpos = priv->display_info.v_center;
 	priv->base.properties.ratio = ((float)priv->display_info.h_resolution / (float)priv->display_info.v_resolution) / 2.0f;
 
+	//setup generic distortion coeffs, from hand-calibration
+	switch (desc->revision) {
```

Comment:
Just abstracting things a little to keep code sane :)
Since this will be part of every driver, and we might want to use the ones from the device if they make sense, it would be logical to put this into a function. Its ok to keep modifying the base struct imo.